plugins/importexport/csv/CSVImportExportPlugin.inc.php

```diff
@@ -93,179 +102,328 @@ function executeCLI($scriptName, &$args) {
+				$publicationFormat = $publicationFormatDao->newDataObject();
+				$publicationFormat->setData("submissionId", $submissionId);
+				$publicationFormat->setData("publicationId", $publicationId);
+				$publicationFormat->setPhysicalFormat(false);
+				$publicationFormat->setIsApproved(true);
+				$publicationFormat->setIsAvailable(true);
+				$publicationFormat->setProductAvailabilityCode('20'); // ONIX code for Available.
+				$publicationFormat->setEntryKey('DA'); // ONIX code for Digital
+				$publicationFormat->setData('name', 'PDF', $submission->getLocale());
+				$publicationFormat->setSequence(REALLY_BIG_NUMBER);
+
+				if ($doi) {
+					$publicationFormat->setStoredPubId('doi', $doi);
+				}
+
+				$publicationFormatId = $publicationFormatDao->insertObject($publicationFormat);
+
+				// Create a publication format date for this publication format.
+				$publicationDate = $publicationDateDao->newDataObject();
+				$publicationDate->setDateFormat('05'); // List55, YYYY
+				$publicationDate->setRole('01'); // List163, Publication Date
+				$publicationDate->setDate($year);
+				$publicationDate->setPublicationFormatId($publicationFormatId);
+				$publicationDateDao->insertObject($publicationDate);
+
+				// Submission File.
+				import('lib.pkp.classes.file.TemporaryFileManager');
+				import('lib.pkp.classes.file.FileManager');
+				import('lib.pkp.classes.core.Core');
+
+				// Submission File.
+				$fileManager = new FileManager();
+				$extension = $fileManager->parseFileExtension($pdfUrl);
+				$dirNames = Application::getFileDirectories();
+				$submissionDir = sprintf(
+					'%s/%d/%s/%d',
+					str_replace('/', '', $dirNames['context']),
+					$press->getId(),
+					str_replace('/', '', $dirNames['submission']),
+					$submissionId
+				);
+
+				$temporaryFileManager = new TemporaryFileManager();
+				$temporaryFilename = $temporaryFileManager->getBasePath() . uniqid() . ".{$extension}";
+				$filePath = getcwd() . "/plugins/importexport/csv/submissionPdfs/" . $pdfUrl;
+				$temporaryFileManager->copyFile($filePath, $temporaryFilename);
+
+				/** @var \PKP\services\PKPFileService */
+				$fileService = Services::get('file');
+				$fileId = $fileService->add(
+					$temporaryFilename,
+					$submissionDir . '/' . uniqid() . '.' . $extension
+				);
+
+				$submissionFile = $submissionFileDao->newDataObject();
+				$submissionFile->setData("submissionId", $submissionId);
+				$submissionFile->setData('uploaderUserId', $user->getId());
+				$submissionFile->setSubmissionLocale($submission->getLocale());
+				$submissionFile->setGenreId($genre->getId());
+				$submissionFile->setFileStage(SUBMISSION_FILE_PROOF);
+				$submissionFile->setAssocType(ASSOC_TYPE_REPRESENTATION);
+				$submissionFile->setData("assocId", $publicationFormatId);
+				$submissionFile->setData("createdAt", Core::getCurrentDate());
+				$submissionFile->setDateModified(Core::getCurrentDate());
+				$submissionFile->setData('mimetype', 'application/pdf');
```

Comment:
Suggest detecting the MIME type from the file rather than hard-coding it as PDF.